test/torchaudio_unittest/functional/torchscript_consistency_impl.py

```diff
@@ -603,46 +579,38 @@ def func(tensor):
             return F.compute_kaldi_pitch(tensor, sample_rate)
 
         tensor = common_utils.get_whitenoise(sample_rate=44100)
-        self._assert_consistency(func, tensor)
+        self._assert_consistency(func, (tensor,))
 
     def test_resample_sinc(self):
         def func(tensor):
             sr1, sr2 = 16000, 8000
             return F.resample(tensor, sr1, sr2, resampling_method="sinc_interpolation")
 
         tensor = common_utils.get_whitenoise(sample_rate=16000)
-        self._assert_consistency(func, tensor)
+        self._assert_consistency(func, (tensor,))
 
     def test_resample_kaiser(self):
-        def func(tensor):
-            sr1, sr2 = 16000, 8000
-            return F.resample(tensor, sr1, sr2, resampling_method="kaiser_window")
-
-        def func_beta(tensor):
-            sr1, sr2 = 16000, 8000
-            beta = 6.0
-            return F.resample(tensor, sr1, sr2, resampling_method="kaiser_window", beta=beta)
-
         tensor = common_utils.get_whitenoise(sample_rate=16000)
-        self._assert_consistency(func, tensor)
-        self._assert_consistency(func_beta, tensor)
+        sr1, sr2 = 16000, 8000
+        beta = 6.0
+        lowpass_filter_width = 6
+        rolloff = 0.99
+        self._assert_consistency(F.resample, (tensor, sr1, sr2, lowpass_filter_width, rolloff, "kaiser_window", None))
+        self._assert_consistency(F.resample, (tensor, sr1, sr2, lowpass_filter_width, rolloff, "kaiser_window", beta))
```

Comment:
Technically out of the scope of this PR but can you use parameterized on `beta`?